src/NzbDrone.Core/DecisionEngine/Specifications/ReleaseRestrictionsSpecification.cs

```diff
@@ -30,37 +30,45 @@ public virtual Decision IsSatisfiedBy(RemoteEpisode subject, SearchCriteriaBase
             _logger.Debug("Checking if release meets restrictions: {0}", subject);
 
             var title = subject.Release.Title;
-            var restrictions = _releaseProfileService.AllForTags(subject.Series.Tags);
+            List<ReleaseProfile> restrictions = _releaseProfileService.AllForTags(subject.Series.Tags);
 
-            var required = restrictions.Where(r => r.Required.IsNotNullOrWhiteSpace());
-            var ignored = restrictions.Where(r => r.Ignored.IsNotNullOrWhiteSpace());
-
-            foreach (var r in required)
+            foreach (ReleaseProfile restriction in restrictions)
             {
-                var requiredTerms = r.Required.Split(new[] {','}, StringSplitOptions.RemoveEmptyEntries).ToList();
+                // TODO: attach Enabled and IndexerId fields to restriction
```

Comment:
I think this is where I might need some help... I took a stab at it, but quickly got overwhelmed.

Reply:
Better to filter that at the start. Probably even better to have releaseProfileService do it by adding a `EnabledForTags(..., subject.Release.IndexerId)` to ReleaseProfileService  or something like it, which would then do:
```
.AllForTags(...)
.Where(v => v.Enabled)
.Where(v => !v.IndexerId.HasValue || v.IndexerId == indexerId)
.ToList()
```

To add it to ReleaseProfile itself you need a database migration, check out the `rename_restrictions_to_release_profiles` migration, you see it rename the table and add a column. You obviously only need to add two columns, bool `Enabled` with default true and int `IndexerId` with default of null (or zero).
Give it migration number 135 and add that to the top of the PR text.

Reply:
Is there something else that needs to be done after specifying the migration to get the columns to be added?

saw this, but not sure if there's some in-process method
https://fluentmigrator.github.io/articles/quickstart.html?tabs=runner-dotnet-fm